script.js

```diff
@@ -19,14 +18,130 @@ function hamburgerAction() {
+    workLanguages: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://kazim110.github.io/',
+    projLink: 'https://github.com/kazim110/My-Portfolio-Project',
+  },
+  {
+    workId: '2',
+    workTitle: 'Multi-Post Stories',
+    workImg: 'snapshoot-portfolio1.png',
+    workDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    workLanguages: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://kazim110.github.io/',
+    projLink: 'https://github.com/kazim110/My-Portfolio-Project',
+  },
+  {
+    workId: '3',
+    workTitle: 'Multi-Post Stories',
+    workImg: 'snapshoot-portfolio2.png',
+    workDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    workLanguages: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://kazim110.github.io/',
+    projLink: 'https://github.com/kazim110/My-Portfolio-Project',
+  },
+  {
+    workId: '4',
+    workTitle: 'Multi-Post Stories',
+    workImg: 'snapshoot-portfolio3.png',
+    workDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    workLanguages: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://kazim110.github.io/',
+    projLink: 'https://github.com/kazim110/My-Portfolio-Project',
+  },
+];
+
+const cards = document.querySelector('.cards');
+
+cards.innerHTML = '';
+
+for (let j = 0; j < workDetails.length; j += 1) {
+  let techLang = '';
+  workDetails[j].workLanguages.forEach((workLanguages) => {
+    techLang = `${techLang}<li>${workLanguages}</li>`;
+  });
+
+  cards.innerHTML += `<div class="card">
+<img class="work-img" src="./img/${workDetails[j].workImg}" alt="Avatar">
+<div class="work-description">
+    <div class="work-title">
+        <h4 class="work-title-main">${workDetails[j].workTitle}</h4>
+        <ul class="work-info">
+            <li class="client">
+                <p>
+                    Canopy
+                </p>
+            </li>
+            <li class="role">
+                <p>
+                    Back End Dev
+                </p>
+            </li>
+            <li class="year">
+                <p>
+                    2015
+                </p>
+            </li>
+        </ul>
+    </div>
+    <div class="work-content-text">
+        <p>
+          ${workDetails[j].workDesc} 
+        </p>
+    </div>
+    <ul class="work-tech-used">
+      ${techLang}
+    </ul>
+    <div class="action-div">
+        <button class="secondary-btn  work-modal-${j}"><p>See Project</p></button>
```

Comment:
- [ ] As requested by the previous reviewer, Please from the project requirements, you should stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=YHGkZAlalTNDuhZx-0) as much as possible. But as you can see you have some differences in the design. 

- [ ] You should add some spacing between the `project details text column` and `the column with tag languages and buttons`

| Your Design |Figma Design |
| :---:   | :---: | 
| ![image](https://user-images.githubusercontent.com/92303891/204068125-a3329461-ca81-442b-9588-003fe6177127.png) | ![image](https://user-images.githubusercontent.com/92303891/204068250-b7b41269-c2dc-4ea4-b6d1-7dc517fe9051.png)|